src/sentry/utils/env.py

```diff
@@ -1,5 +1,16 @@
 import sys
 
+from django.conf import settings
 
-def in_test_environment():
+
+def in_test_environment() -> bool:
     return "pytest" in sys.argv[0] or "vscode" in sys.argv[0]
+
+
+def is_split_db() -> bool:
+    if len(settings.DATABASES) != 1:  # type: ignore
+        return True
```

Comment:
In saas, this condition will alwasy be true as our production DB is spread across multiple hosts.